### PR TITLE
changed from gson to jackson

### PR DESCRIPTION
**Pull request by FaintSmileVL**

### CODE REVIEW

File: src/main/java/ru/finex/evolution/impl/MigrationDao.java [hunk 1]
```diff
@@ -23,7 +24,7 @@
 @SuppressWarnings({"checkstyle:MissingJavadocMethod", "checkstyle:Indentation"})
 public class MigrationDao {
 
-    private final Gson gson = new Gson();
+    ObjectMapper mapper = new ObjectMapper();
```
Comment (zcxv) [inline src/main/java/ru/finex/evolution/impl/MigrationDao.java]: Missing `private final`

File: pom.xml [hunk 2]
```diff
@@ -17,7 +17,7 @@
         <delombok.source>${project.build.directory}/delombok-sources</delombok.source>
 
         <lombok.version>1.18.20</lombok.version>
-        <gson.version>2.8.6</gson.version>
+        <jackson.version>2.12.5</jackson.version>
```
Comment (zcxv) [inline pom.xml:20]: GSON already connected at 92 line. 
Jackson doesnt defined, define it like:
```
<dependencyManagement>
    ...
    <dependency>
        <groupId>com.fasterxml.jackson</groupId>
        <artifactId>jackson-bom</artifactId>
        <version>${jackson.version}</version>
        <type>pom</type>
        <scope>import</scope>
    </dependency>
</dependencyManagement>

<dependencies>
    ...
    <dependency>
        <groupId>com.fasterxml.jackson.core</groupId>
        <artifactId>jackson-core</artifactId>
    </dependency>
    <dependency>
        <groupId>com.fasterxml.jackson.core</groupId>
        <artifactId>jackson-databind</artifactId>
    </dependency>
</dependencies>
```

Comment (FaintSmileVL) [reply pom.xml:20]: Now fixed

File: src/main/java/ru/finex/evolution/impl/MigrationDao.java (hunk 1 above)
Comment (FaintSmileVL) [reply src/main/java/ru/finex/evolution/impl/MigrationDao.java]: added

**Pull request merged**